Decred Wallet/Features/Wallet Setup/WalletSetupViewController.swift

```diff
@@ -27,4 +27,17 @@ class WalletSetupViewController: WalletSetupBaseViewController {
         let netType = BuildConfig.IsTestNet ? "testnet" : BuildConfig.NetType
         build?.text = "build \(netType) " + dateformater.string(from: AppDelegate.compileDate)
     }
+    
+    // MARK:- Prepare for segue
+
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+
```

Comment:
The code can probably still be readable without this extra line here. Thoughts?

Reply:
I happen to have more breathing room in my code. I will get rid of the extra line since compact is the current setup.